Sources/ApolloCodegenLib/Templates/FragmentTemplate.swift

```diff
@@ -1,24 +1,25 @@
-struct FragmentTemplate {
+import Foundation
 
+/// Provides the format to convert a [GraphQL Fragment](https://spec.graphql.org/draft/#sec-Language.Fragments)
+/// into Swift code.
+struct FragmentTemplate: TemplateRenderer {
+  /// IR representation of source [GraphQL Fragment](https://spec.graphql.org/draft/#sec-Language.Fragments).
   let fragment: IR.NamedFragment
+  /// IR representation of source GraphQL schema.
   let schema: IR.Schema
-  let config: ApolloCodegenConfiguration.FileOutput
 
-  func render() -> String {
+  var target: TemplateTarget { .operationFile }
```

Comment:
If a fragment is being shared across operations and those operations are in separate modules, the Fragment would need to be include in the schema module. We probably need to put some more thought into this.